modules/swagger-jaxrs2/src/test/java/io/swagger/jaxrs2/annotations/parameters/ParametersTest.java

```diff
@@ -98,7 +110,21 @@ public void testSimpleCallback() {
                                         format = "id",
                                         description = "the generated id",
                                         readOnly = true
-                                ))
+                                )),
+                        @Parameter(in = "query", name = "arrayParameter", required = true, explode = Explode.TRUE,
+                                array = @ArraySchema(maxItems = 10, minItems = 1,
+                                        schema = @Schema(implementation = ParametersTest.SubscriptionResponse.class),
+                                        uniqueItems = true
+                                ),
+                                schema = @Schema(
+                                        type = "int",
+                                        format = "id",
+                                        description = "the generated id",
+                                        readOnly = true),
+                                content = @Content(schema = @Schema(type = "int",
+                                        format = "id",
+                                        description = "the generated id",
+                                        readOnly = true)))
```

Comment:
what should be taking precedence (array or content or schema)? did you discuss this with @webron?

Reply:
https://github.com/swagger-api/swagger-core/pull/2416 Suggested by Ron in this PR.